tasks/generateSettings.js

```diff
@@ -27,6 +28,7 @@ const legacySettingMap = {
   ImportNotification: 'importNotification',
   PrintingImprovements: 'printingImprovements',
   RemovePositiveHighlight: 'removePositiveHighlight',
+  ResizeInspector: 'resizeInspector',
```

Comment:
Looks like this snuck in from the other PR, but that's fine, shouldn't hurt anything.